### ISSUE TITLE
Not found grpc and protobuf extensions

### ISSUE THREAD

**Issue by pepebotika69** [khepin/php-grpc-server-notes#7]
Hi! The work you have done is great, the best example I found in the internet.

So, this is more like a question then an issue:

The changes I have made:

- I changed the docker-compose.yaml to 3.7
- I changed php version to 8.0

I did this just because in my project we use this versions, I dont think that this can give us some problem.

Anyway, for service simplecache we have this docker-file:
```
# Dockerfile-app
FROM php:7.4-cli

# Extensions
RUN echo starting && \
    docker-php-ext-enable grpc && \
    docker-php-ext-enable protobuf
```

When I tried to up the container I got an error: no grpc extension found? and later no protobuf found.
So I added:
```
RUN apt-get update
RUN apt-get install libz-dev
RUN pecl install grpc
RUN pecl install protobuf
```

And the container run.

As I told you, I am not sure that this is an issue in documentation, is more like a question:

Are this lines needed?
```
RUN apt-get update
RUN apt-get install libz-dev
RUN pecl install grpc
RUN pecl install protobuf
```

**Comment by khepin** [comment]
Yes, you are correct. This was due to an issue with my setup at the time.
Can you make a PR with those updates?

Thank you

**Comment by pepebotika69** [comment]
yes, of course I will

**Comment by khepin** [comment]
Thanks, merged it
